src/com/jwetherell/algorithms/strings/StringFunctions.java

```diff
@@ -13,6 +13,12 @@ public class StringFunctions {
     private static final char SPACE = ' ';
 
     public static final String reverseWithStringConcat(String string) {
+        if (string == null) {
```

Comment:
Check for string null to confirm string exists, returning and empty string if it is null.